end-to-end-tests-localdb/docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2016 The Hyve B.V.
```

Comment:
@pvannierop would be good here to use a specific version of the backend image now that it's on docker hub e.g. `3.0.1`

Reply:
then u can prolly remove this Dockerfile

Reply:
There are two issues related to this that I see as a problem:

1. my Dockerfile allows you to package a specific frontend version into the portal image.
```
RUN mvn -DskipTests clean install **$MAVEN_OPTS** \
	&& unzip portal/target/cbioportal*.war -d $CATALINA_HOME/webapps/cbioportal \
	&& mv scripts/target/scripts-*.jar /root/ \
	&& mvn clean \
	&& mkdir scripts/target/ \
	&& mv /root/scripts-*.jar scripts/target/
```
where MAVEN_OPTS is something like
```
-Dfrontend.version=js737ghd -Dfrontend.groupId=cm.github.thhyve
```
So, if we can update the official cbioportal Dockerfile in the same manner I can do this. Do you think this is something that can be incorporated?

2. I pass in the address of the session service container. The current Docker file does not seem to allow for this?

Reply:
@pvannierop 

1. I don't think for this PR it's necessary to have a specific frontend version included. I would expect it to use the frontend in the repo

2. You can specify the address of the session service like this:

https://github.com/knowledgesystems/knowledgesystems-k8s-deployment/blob/master/cbioportal/cbioportal_spring_boot.yaml#L101

Reply:
You can build an image for a specific branch like this:

```
docker build https://github.com/cBioPortal/cbioportal.git#master -f docker/web-and-data/Dockerfile
```

I think we'll try and set something up that will auto build the master, rc and release branch images. So you would only have to use the above to build your own image if that's not available. Maybe some documentation on how to build your own image and then passing the `BACKEND` variable with that tag is sufficient. I think the more common use case is using latest `master`, `rc` or `release-x.y.z` branch. Those images you can assume that one can pull them from docker hub

Reply:
I've set up automated builds now for master and rc. Tag is just `master` and `rc`.

https://github.com/cBioPortal/cbioportal-frontend/pull/2408

Reply:
> You can specify the address of the session service like this:

> https://github.com/knowledgesystems/knowledgesystems-k8s-deployment/blob/master/cbioportal/cbioportal_spring_boot.yaml#L101

Great. How can I apply this parameter to the cBioPortal container? Is this passed with the run command? How?

Reply:
> You can build an image for a specific branch like this:

> docker build https://github.com/cBioPortal/cbioportal.git#master -f docker/web-and-data/Dockerfile

Ah nice, I will integrate this.